tabernacle/src/routes/leagueSession/PointsModal.jsx

```
@@ -56,18 +66,27 @@ export default function PointsModal({ isOpen, closeModal, selected }) {
                   className="mb-2 text-2xl font-medium leading-6 text-gray-900 flex flex-wrap md:no-wrap justify-center md:justify-between"
                 >
                   <span className="text-2xl font-bold">
-                    {selected?.participant}
+                    {selected?.participant}{" "}
+                    {decklist?.url && (
+                      <span className="text-base hover:text-sky-400">
+                        -{" "}
+                        <Link to={decklist.url}>
+                          {decklist.name} // {decklist.code}
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)